annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -2,5 +2,5 @@ spring.batch.job.enabled=false
 genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
 genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=summary
+genomenexus.enrichment_fields=fields=hotspots,mutation_assessor
```

Comment:
the double equals sign looks risky, but I guess it is ok if the properties file parser breaks the key/value at the first equals sign. so the URL below is going to look like .............&fields=hotspots,mutation_assessor
which makes more sense than .........&summary=summary

Reply:
Thanks!